docs/deployment/rds-s3/README.md

```diff
@@ -138,6 +138,72 @@ Resources:
 
     kubectl apply -f https://raw.githubusercontent.com/aws/secrets-store-csi-driver-provider-aws/main/deployment/aws-provider-installer.yaml
    ```
+   
+   3. [Optional] If you prefer to use a different secret name for RDS or S3, patch files exist under the overlay at `distributions/aws/aws-secrets-manager/overlays/configurable-secrets`. In the overlay folder, replace the default secret names in the files `deployment_patch.yaml` and `secrets_manager_patch.yaml`. 
```

Comment:
Since you are changing the readme, do you mind pulling in #78 #89 and making your changes on top of it?